support-frontend/app/services/stepfunctions/SupportWorkersClient.scala

```diff
@@ -179,8 +179,12 @@ class SupportWorkersClient(
           request.headers.get("X-Forwarded-For").flatMap(_.split(',').headOption).getOrElse(request.remoteAddress),
       )
       isExistingAccount = createPaymentMethodState.paymentFields.left.exists(_.isInstanceOf[ExistingPaymentFields])
+      name =
+        (if (user.isTestUser) "TEST-" else "") +
```

Comment:
Maybe a prefix of `TestUser` might be a (🐭 tiny) more descriptive?

Reply:
it might, I was thinking of using `T-` to make it even shorter, but I'm not too fussed either way as it only appears for test users
```suggestion
        (if (user.isTestUser) "TestUser-" else "") +
```